components/usage/pkg/scheduler/reset_usage_job.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package scheduler
+
+import (
+	"time"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+)
+
+func NewResetUsageJobSpec(schedule time.Duration) (JobSpec, error) {
```

Comment:
The constructor is called `NewResetUsageJobSpec` but the type is simply `ResetUsageJob`. Do you want to make them consistent?

Reply:
done